canonical-web-and-design/charmhub.io
# Get featured charms from API

## Done
Replaced hard-coded featured charms on homepage with charms from the API

## QA
- Run site locally with `dotrun`
- Make sure your `.env.local` file contains:
```
# Staging API for publisher:
CHARMSTORE_PUBLISHER_API_URL=https://api.staging.charmhub.io/
CANDID_API_URL=https://api.staging.jujucharms.com/identity/

# Staging API for consumer:
CHARMSTORE_API_URL=https://api.staging.snapcraft.io/
```

- Go to http://localhost:8045
- Check that there are featured charms on the page
- Check that the search and filters still work

## Issue
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1841
